packages/flutter/test/rendering/rendering_tester.dart

```diff
@@ -19,7 +21,15 @@ class TestRenderingFlutterBinding extends BindingBase with SchedulerBinding, Ser
   /// while drawing the frame. If [onErrors] is null and [FlutterError] caught at least
   /// one error, this function fails the test. A test may override [onErrors] and
   /// inspect errors using [takeFlutterErrorDetails].
-  TestRenderingFlutterBinding({ this.onErrors });
+  ///
+  /// Errors caught between frames will cause the test to fail unless
+  /// [FlutterError.onError] has been overridden.
```

Comment:
I wonder if we could add a teardown here like we do in the widgets test binding, and check that `FlutterError.onError` is still the one we want it to be. WDYT?

Reply:
flutter_test doesn't add a tearDown, does it? I think it just wraps `test` and tests it there. I'd rather burn `tearDown` to the ground and remove it entirely, it always causes me confusion...